### ISSUE TITLE
Billed "storage used" vs. actual used space incorrect.

### ISSUE THREAD

**Issue by Th3Van** [Storj/bridge-gui#162]
A few weeks ago, I did a +24h upload "stress-test", uploading ~170.000 x 384MB files, resulting in close to 70TB being uploaded to a single bucket. (I still have to crunch the numbers in the log files -  but more on that later in #stress-testers)

Thing is, about a week after preforming the test, I deleted the bucket, re-created it again using the same name and it seems to be empty. But for unknown reason the daily $33 still seems to be added to my billing history and balance, even tho the bucket with all the test data has been deleted about two weeks ago...

Yesterday I re-created the bucket, throw a few files into it, and now I'm waiting for the the billing system to update. Hopefully updating my `storage used` to the correct amount of GB as shown below. I'll post the result tomorrow. (none of the buckets are holding more than 2000 files, btw)

**Buckets :**
```
root@server007:~# storj list-buckets
ID: e09d7ffc78da193eb45897b6    Decrypted: true         Created: 2017-06-13T22:54:56.911Z       Name: S_n_K
ID: 07b77c7417539c0b9002a5e9    Decrypted: true         Created: 2017-06-24T13:17:39.249Z       Name: S_n_K_Take-2_-_24-06-2017
ID: a01ac0afd9a6fd95ee243018    Decrypted: true         Created: 2017-07-14T23:37:44.439Z       Name: test2
ID: b126caea127b0c3278ada1a7    Decrypted: true         Created: 2017-07-23T23:42:06.429Z       Name: Test123  (Used for the 24h upload-test)
ID: f6eece9e9945a0ab5b018f97    Decrypted: true         Created: 2017-07-24T02:40:48.722Z       Name: test3
```
**Bucket Sizes :**
```
root@server007:~# storj list-files e09d7ffc78da193eb45897b6 | awk '{sum+=$4}END{printf "Total for this bucket: %.f Bytes\n",sum}'
Total for this bucket: 13.470.156.275 Bytes

root@server007:~# storj list-files 07b77c7417539c0b9002a5e9 | awk '{sum+=$4}END{printf "Total for this bucket: %.f Bytes\n",sum}'
Total for this bucket: 15.993.965.360 Bytes

root@server007:~# storj list-files a01ac0afd9a6fd95ee243018 | awk '{sum+=$4}END{printf "Total for this bucket: %.f Bytes\n",sum}'
Total for this bucket: 10.737.418.240 Bytes

root@server007:~# storj list-files b126caea127b0c3278ada1a7 | awk '{sum+=$4}END{printf "Total for this bucket: %.f Bytes\n",sum}'
Total for this bucket: 40.115.315.192 Bytes

root@server007:~# storj list-files f6eece9e9945a0ab5b018f97 | awk '{sum+=$4}END{printf "Total for this bucket: %.f Bytes\n",sum}'
Total for this bucket: 0 Bytes
```
**Total sum of data in five buckets : 80.316.855.067 Bytes (2.6GB / day = $0.039 / day)**

 
![billing - storage- th3van](https://user-images.githubusercontent.com/18688983/28532206-7070d97a-7099-11e7-8b7f-71c7f1eec3ec.png)

//Th3Van

**Comment by Th3Van** [comment]
For the last couple of days I have been digging a bit more into this issue, and discovered this : 

* **Jul 28, 2017** - Removed all my buckets, and was charged **$33.7571** for storage used (which equals to 70TB vs. the actually 80GB of storage I was using). Since I was using storage that particularly day, it was ok to charge me for it.
* **Jul 29, 2017** - Things seems to be working as it should! Not being charged at all, since no buckets existed.
* **Jul 30, 2017** - Using Libstorj, I created the bucket again, with the same name as the bucket I deleted Jul 28 : `Test1234` and waited until the billing system did its turn.
Surprisingly, even though having a bucket with the same name (and ID) containing some test files, I was (again) being charged as if the bucket was storing 70TB of files in it @ **$33.7050** every day, which of course are incorrect.

In short : If you delete a bucket with files in it and re-creates it again, using the exact same name/ID, you could end up being billed for files you are no longer in control of... like some sort of ghost files.. :)

I'll look more into this and report back..

![billede](https://user-images.githubusercontent.com/18688983/28758396-5b4a72b0-7597-11e7-8457-bef1b31d5474.png)
